Lib/glyphsLib/builder.py

```diff
@@ -463,6 +463,16 @@ def set_custom_params(ufo, parsed=None, data=None, misc_keys=(), non_info=()):
         if name in ('openTypeOS2WeightClass', 'openTypeOS2WidthClass'):
             value = int(value)
 
+        if name == 'Replace Feature':
+            if len(value.split(';')) > 2:
+                tag = value.split(';')[0]
+                code = ';'.join(value.split(';')[1:])
+                replace_dict = {'feature': tag, 'code': code}
+                new_code = 'feature %(feature)s {\n %(code)s \n} %(feature)s;' % replace_dict
+                pattern = re.compile('feature\s*%(feature)s[^;].*%(feature)s\s*;' % replace_dict, re.DOTALL)
+                replaced = re.sub(pattern, new_code, ufo.features.text)
```

Comment:
minor: you could also directly use the `sub` method of the compiled pattern object
```diff
- replaced = re.sub(pattern, new_code, ufo.features.text)
+ replaced = pattern.sub(new_code, ufo.features.text)
```